### PR TITLE
fix: Fix license headers

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
fix: Fix license headers

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/19946

### How to test this PR?

```bash
cat EOF << > /tmp/patch.yaml
<PATCH_CONTENT>
EOF

chectl server:deploy \
     --installer operator \
     --platform <PLATFORM_TO_DEPLOY> \
     --che-operator-image <CUSTOM_OPERATOR_IMAGE> \
     --che-operator-cr-patch-yaml /tmp/patch.yaml
```

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [Custom resource definition file is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#updating-custom-resource-definition-file)
- [ ] [OLM bundles are up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#update-olm-bundle)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by tolusha — APPROVED** [review]
Check release scripts

File: .ci/cico_updates_openshift.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2020 Red Hat, Inc.
+# Copyright (c) 2019-2021 Red Hat, Inc.
```
Comment (mmorhun) [inline .ci/cico_updates_openshift.sh:3]: We lose original creation date

File: .github/bin/common.sh [hunk 2]
```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 #
-# Copyright (c) 2012-2021 Red Hat, Inc.
+# Copyright (c) 2019-2021 Red Hat, Inc.
```
Comment (mmorhun) [inline .github/bin/common.sh:3]: The same here

File: .ci/cico_updates_openshift.sh (hunk 1 above)
Comment (AndrienkoAleksandr) [reply .ci/cico_updates_openshift.sh:3]: Sorry, but I thought we considered in the stand up to align all license headers to the same value, but after that developers should care about correct year for new files.

**Pull request merged**